charts/sn-platform/templates/broker/broker-statefulset.yaml

```diff
@@ -297,6 +297,7 @@ spec:
           {{- include "pulsar.broker.runtime.volumeMounts" . | nindent 10 }}
           {{- include "pulsar.broker.offload.volumeMounts" . | nindent 10 }}
           {{- include "pulsar.function.worker.config.volumeMounts" . | nindent 10 }}
+          {{- include "pulsar.broker.offload.filesystem.config.volumeMounts" . | nindent 10 }}
```

Comment:
If filesystem is set to `false`, do you still need to include this mount?

Reply:
no, already handle that in the broker.tpl